Dockerfile

```diff
@@ -23,4 +23,4 @@ ADD .travis.yml .
 ADD .rubocop.yml .
 ADD test /app/test
 
-CMD bundle exec bin/secrets
+CMD mkdir -p /secrets/bin && cp /app/bin/wait_for_it.sh /secrets/bin && bundle exec bin/secrets
```

Comment:
yeah secrets is added to the completed image ... but that still leaves the race condition on startup since the script might not be ready ... so host app has to wait for the script to appear ... might as well wait for .done to appear ?